ui/html/ui-tools.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>Lamassu Machine Dev Tools</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <style>
+    body {
+      background-color: #82dda3;
```

Comment:
Don't know what was thinking when I committed this color. It is too harsh for eyes. Lighter green would look better

```suggestion
      background-color: #beeecf;
```